src/test/scala/org/broadinstitute/dsde/rawls/jobexec/SubmissionSpec.scala

```diff
@@ -345,6 +345,11 @@ class SubmissionSpec(_system: ActorSystem) extends TestKit(_system) with FlatSpe
     assert(submissionStatusResponse.notstarted(0).errors(0).value == "Expected single value for workflow input, but evaluated result set was empty")
     assert(submissionStatusResponse.notstarted(1).errors(0).value == "Unable to submit workflow when creating submission")
     assert(submissionStatusResponse.notstarted(2).errors(0).value == "Unable to submit workflow when creating submission")
+
+    // all of these workflows fail, ensure the submission status is done
+    assertResult(SubmissionStatuses.Done) {
+      submissionStatusResponse.status
+    }
```

Comment:
there is already a test for that that still works as expected, the one thing it doesn't do is verify that failures of this type along with successful Workflows together returns submitted, but that seemed equivalent to me of my test and the other test since I have tested that notstarted is concatenated properly